robots/configBasedRobot.py

```diff
@@ -29,15 +34,57 @@ def __init__(self, period: float = 0.02) -> None:
 
         self.navx = navx._navx.AHRS.create_spi()
 
-    def getAutonomousCommand(self):
-        return(Autonomous(self.driveTrain, self.navx))
+        # We should only need to do this once
+        wpilib.SmartDashboard.putString("Robot Version", self.getDeployInfo("git-hash"))
+        wpilib.SmartDashboard.putString("Git Branch", self.getDeployInfo("git-branch"))
+        wpilib.SmartDashboard.putString(
```

Comment:
consider:  creat a new network table for versions and config info. Seems like we should get better at organizing our data